### ISSUE TITLE
Create the Login page in React

### ISSUE THREAD

**Issue by Sanyam2000-dot** [ADRE9/bunk-manager-mern#142]
**Deliverables:**
Add the deliverables with their description here.

- [ ] Link for design is [https://www.figma.com/file/BBOPzTwcLPLc8B3HckxQqH/Login?node-id=0%3A1](url)
- [ ] Use Material UI
- [ ] Make for both light and dark theme
![Screenshot (240)](https://user-images.githubusercontent.com/53525430/117053660-d8ad3000-ad36-11eb-8b80-85b73b66ffc2.png)

**Comment by shobit1337** [comment]
Hey @Sanyam2000-dot , I did like to work on the login page. I'm working with react and Material UI already, That's why I want to build this one, Thanks

**Comment by Sanyam2000-dot** [comment]
> Hey @Sanyam2000-dot , I did like to work on the login page. I'm working with react and Material UI already, That's why I want to build this one, Thanks

Go ahead! I'm assigning this issue to you. If you face any doubts please drop the message in comment only and keep us updated about your progress. Happy Coding :)

**Comment by Sanyam2000-dot** [comment]
Hey @shobit1337, Can you update me about your progress?

**Comment by shobit1337** [comment]
> Hey @shobit1337, Can you update me about your progress?

Yea I am currently understanding the codebase of the project, there were some packages that were used and new to me.
Also on my computer, I'm unable to run the server it was asking for some env file that was missing, So, right now only the client-server is running so working from that.
I will update you by Monday for sure @Sanyam2000-dot .

**Comment by Sanyam2000-dot** [comment]
No issues take your time :)

**Comment by mn-gt** [comment]
Can I also work on this issue?
As I wanted to learn react.Js by designing this..

**Comment by Sanyam2000-dot** [comment]
Hey! @shobit1337 Any progress so far?

**Comment by shobit1337** [comment]
> Hey! @shobit1337 Any progress so far?

Yeah working on it... I have separately designed the header and auth card component. Need to complete the get in touch and footer section then just put it together and make a pull request for review.
Sorry for the delay. My father has been tested positive 2 days ago, Hence only able to give 1hr a day and learning side by side too.

**Comment by kunaljain0212** [comment]
No issues, take your time just keep us posted. Also, take care of your family! @shobit1337

**Comment by shobit1337** [comment]
I won't be able to continue my work on this issue due to health issue. Sorry guys, You can assign it to someone else.

**Comment by mn-gt** [comment]
Can you please assign to me this task  @kunaljain0212 ?

**Comment by Sanyam2000-dot** [comment]
> I won't be able to continue my work on this issue due to health issue. Sorry guys, You can assign it to someone else.

No worries. Take care!

**Comment by Sanyam2000-dot** [comment]
> Can you please assign to me this task @kunaljain0212 ?

Go ahead. I'm assigning this to you.

**Comment by ADRE9** [comment]
Any updates @mn-gt ?

**Comment by mn-gt** [comment]
I am working on this...

**Comment by mn-gt** [comment]
Hey! @ADRE9 @Sanyam2000-dot 
I have almost completed the login part but there are some problems in UI  so I want some more time..

**Comment by ADRE9** [comment]
Take your time.

**Comment by mn-gt** [comment]
Hey! @ADRE9 @Sanyam2000-dot 
1. Can anyone please clarify the theme color details (For light and dark mode ) and font family (if any change required). 
2.  Will the app bar be visible after logged the user/filling sign up form (please provide any screenshot, if any)?

**Comment by ADRE9** [comment]
Check out the figma file it has the colors specified. And yes the app bar will be visible after login . The figma file has been provided at the top.
